lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/AccessControlRulesParser.java

```diff
@@ -18,19 +18,21 @@
 
```

Comment:
What is the motivation behind this commit?

Reply:
I squashed all my changes into one commit to not have inconsistent states in between.